mappings/net/minecraft/entity/passive/LlamaEntity.mapping

```diff
@@ -19,3 +19,7 @@ CLASS arv net/minecraft/entity/passive/LlamaEntity
 		ARG 0 color
 	METHOD u setVariant (I)V
 	METHOD v setStrength (I)V
+	METHOD h spitAt (Laio;)V
+		ARG 1 target
+	METHOD a breedWith (Laid;)Larv;
```

Comment:
Looks like `createChild` to me by param and return types.

Reply:
There is already a different function literally called `createChild`, which is used in this function. Also, there is a function `canBreedWith`.

Reply:
what does this do in addition to calling create child?

Reply:
createChild gives a plain new Entity instance. This function, however, does the inherit-stats-from-parents thing and gives the child the color of one of the parents. One parent is the instance, the function is called on and the other is the parameter.